app/src/main/res/drawable/ic_skip_login.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_skip_login` appears to be unused
 Unused resources make applications larger and slow down builds.